src/frontend/src/lib/components/address-book/ContactForm.svelte

```diff
@@ -20,16 +20,14 @@
 
 	const assertValid = (): boolean => notEmptyString(trimmedName) && !isNameTooLong;
 
-	const isValid = $derived(assertValid());
+	export const isValid = () => assertValid();
```

Comment:
Related to my previous comment, I would expect `isValid` to become `$bindable()` or use a more complex pattern - like a context or shared stated exposed in a class or function - to make the property reactive.